stories/Megamenu/Megamenu.stories.js

```diff
@@ -0,0 +1,509 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+import { withKnobs } from '@storybook/addon-knobs/react'
+
+import {
+  Megamenu,
+  LinkList,
+  LinkListItem,
+  Navbar,
+  Icon,
+  Row,
+  Col
+} from '../../src'
+
+import Classic from './docs/Classic.md'
+import ClassicWithMore from './docs/ClassicWithMore.md'
+import ClassicWithHeader from './docs/ClassicWithHeader.md'
+import ClassicWithHeaderAndMore from './docs/ClassicWithHeaderAndMore.md'
+
+// import MegamenuExample from './MegamenuExample'
+
+const Classico = () => {
+  return (
+    <Navbar expand="lg">
```

Comment:
I think the `Navbar` component should not be used here. The `Megamenu` should be enough on its own.